src/main/java/dinkplugin/domain/SeasonalPolicy.java

```diff
@@ -0,0 +1,17 @@
+package dinkplugin.domain;
+
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+public enum SeasonalPolicy {
+    ACCEPT("Notify normally"),
+    REJECT("Drop notification"),
+    FORWARD_TO_LEAGUES("Use Leagues URL");
```

Comment:
```suggestion
    ACCEPT("Notify normally"),
    FORWARD_TO_LEAGUES("Use Leagues URL");
    REJECT("Off"),
```
maybe this and change the description from `suppress` to `send`?